Classes/DataProcessing/LanguageMenuProcessor.php

```diff
@@ -0,0 +1,247 @@
+                        'stdWrap.' => [
+                            'wrap' => '"language":|'
+                        ]
+                    ],
+                    '72' => 'USER',
+                    '72.' => [
+                        'userFunc' => 'BK2K\BootstrapPackage\DataProcessing\LanguageMenuProcessor->getLocaleAsJson',
+                        'userFunc.' => [
+                            'languageUid' => '{register:languageUid}'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => '"locale":|'
+                        ]
+                    ],
+                    '73' => 'USER',
+                    '73.' => [
+                        'userFunc' => 'BK2K\BootstrapPackage\DataProcessing\LanguageMenuProcessor->getHreflangAsJson',
+                        'userFunc.' => [
+                            'languageUid' => '{register:languageUid}'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => '"hreflang":|'
+                        ]
+                    ],
+                    '74' => 'USER',
+                    '74.' => [
+                        'userFunc' => 'BK2K\BootstrapPackage\DataProcessing\LanguageMenuProcessor->getDirectionAsJson',
+                        'userFunc.' => [
+                            'languageUid' => '{register:languageUid}'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => '"direction":|'
+                        ]
+                    ],
+                    '99' => 'RESTORE_REGISTER'
+                ]
+            );
+        }
+    }
+
+    /**
+     * Gets the data of the current record in JSON format
+     *
+     * @param string $content
+     * @param array $conf
+     * @return string JSON encoded data
+     */
+    public function getLanguageTitleAsJson($content, $conf)
+    {
+        return $this->jsonEncode($this->cObj->data);
+    }
+
+    /**
+     * Gets the data of the current record in JSON format
+     *
+     * @param string $content
+     * @param array $conf
+     * @return string JSON encoded data
+     */
+    public function getLanguageAsJson($content, $conf)
+    {
+        return $this->jsonEncode($this->cObj->data);
+    }
+
+    /**
+     * Gets the data of the current record in JSON format
+     *
+     * @param string $content
+     * @param array $conf
+     * @return string JSON encoded data
+     */
+    public function getLocaleAsJson($content, $conf)
+    {
+        return $this->jsonEncode($this->cObj->data);
+    }
+
+    /**
+     * Gets the data of the current record in JSON format
+     *
+     * @param string $content
+     * @param array $conf
+     * @return string JSON encoded data
+     */
+    public function getHreflangAsJson($content, $conf)
+    {
+        return $this->jsonEncode($this->cObj->data);
+    }
+
+    /**
+     * Gets the data of the current record in JSON format
```

Comment:
Can you please drop these descriptions? They say nothing more than the method names - in fact they say less. They have to go :)

Reply:
same as before ;)